sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetCommitterSuite.scala

```diff
@@ -116,7 +116,7 @@ class ParquetCommitterSuite extends SparkFunSuite with SQLTestUtils
   test("SPARK-48804: Fail fast on unloadable or invalid committers") {
     Seq("invalid", getClass.getName).foreach { committer =>
       val e = intercept[IllegalArgumentException] {
-        withSQLConf(SQLConf.PARQUET_OUTPUT_COMMITTER_CLASS.key -> committer)()
+        withSQLConf(SQLConf.PARQUET_OUTPUT_COMMITTER_CLASS.key -> committer)(())
```

Comment:
cc @yaooqinn 

Is it more appropriate here to keep an empty function or to change to manually calling `SQLConf.get.setConfString`?

Reply:
Using `setConf` might need to unset the config to prevent abnormal cases

Reply:
https://github.com/apache/spark/blob/9bff2c8bc5059f5be0dc6e8105c11403942a0b9f/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/SQLConfHelper.scala#L38-L62


From the code, if the `conf.setConfString(k, v)` on line 54 throws an exception, the current configuration will not be reset. Am I  right?

Reply:
Resetting won't be necessary for errors but will be required for success (which is abnormal here).

Reply:
Anyway, the current fix is ok for me, although this approach seems more like "having to define an empty function to fix a compilation warning." 

Also, 
```
withSQLConf(SQLConf.PARQUET_OUTPUT_COMMITTER_CLASS.key -> committer) {}
```
 is ok too.

Reply:
I have already `grep` the logs of GA, and after this PR, the `related logs` have been eliminated.